runner/test/cases/server/plugins/engine/components/TextField.test.ts

```diff
@@ -27,9 +27,9 @@ suite("TextField", () => {
     };
 
     const component = new TextField(componentDefinition, formModel);
-
+ 
```

Comment:
could you have a crack at adding a test for a component configured with some regex?